Dockerfile-env

```diff
@@ -1,10 +0,0 @@
-FROM centos/python-36-centos7
```

Comment:
We need to keep Dockerfile-env for the docker-compose dev flow.

Reply:
Okay. 

Just be aware that this means the docker-compose workflow users are now responsible for tracking any changes to the Dockerfile made upstream now that the OpenShift workflow no longer uses a custom Dockerfile.

Reply:
Fixed in a730244